alloc-customization-point.html

```diff
@@ -63,21 +63,24 @@ <h2>tl;dr</h2>
 </p>
 
 <pre><blockquote><code>template &lt;class _Tp, class _Alloc&gt;
-  struct __domain_allocator_traits {
+ struct __domain_allocator_traits {
     static constexpr bool value = false;
   };
 </code></blockquote></pre>
 
+<p>__domain_allocator_traits&lt;class T, class Alloc&gt;::value specifies whether type T supports uses allocator construction with a 
+domain specific allocator DomainAlloc which can be mapped to an object of Alloc type. A specialization is required to provide an Alloc to DomainAlloc 
```

Comment:
I think it's mapped from, rather than mapped to, or perhaps created from.

Reply:
Also, a specialization is *not* required to provide the conversion function, if the specialization's ::value is false.

Reply:
Replaced mapped to with created from. 
Modified the sentence to "A specialization is required to provide an Alloc to DomainAlloc 
convert function if the specialization's ::value is true."